Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/ConceptTensorFlowObjectDetectionWebcam.java

```diff
@@ -0,0 +1,153 @@
+     * 'parameters.vuforiaLicenseKey' is initialized is for illustration only, and will not function.
+     * A Vuforia 'Development' license key, can be obtained free of charge from the Vuforia developer
+     * web site at https://developer.vuforia.com/license-manager.
+     *
+     * Vuforia license keys are always 380 characters long, and look as if they contain mostly
+     * random data. As an example, here is a example of a fragment of a valid key:
+     *      ... yIgIzTqZ4mWjk9wd3cZO9T1axEqzuhxoGlfOOI2dRzKS4T0hQ8kT ...
+     * Once you've obtained a license key, copy the string from the Vuforia web site
+     * and paste it in to your code on the next line, between the double quotes.
+     */
+    private static final String VUFORIA_KEY =
+            "AQYaetf/////AAABmTKBOnYV+03Dsg0qldOiWBJtNgHX1kGW8jNu7X1SH7uDsyjR8VKoRbCeY5QE5OArcAm/vmSbxm" +
+                    "/fUO7ImfFpyNDYUGMG6vCXg/aLnGdT6Obt380jnlTQJwYGhJv1J3ULYK4J07HrtN+T6nGUfF0XTBo07gHQN" +
+                    "a6QU0Pc4P2qmu3/l25K1Tm7P4Fjf2NXvk0AR29y37IP0MOSJJ7/oMcDDs/vBxvEwjXo5ycvHyuzqieNcK7V" +
+                    "R9gOBYb2fpC246GZD7GykaILZoqU3bDTw0Ofcssc3OxnjQJP8DTgsqpJxwXG0DVrfWF3eYFI5aXNQozmoE" +
+                    "JYRAA4iLesJMQI58VVzz2R+frnTs8vyznlTzstxsO6";
+
+    /**
+     * {@link #vuforia} is the variable we will use to store our instance of the Vuforia
+     * localization engine.
+     */
+    private VuforiaLocalizer vuforia;
+
+    /**
+     * {@link #tfod} is the variable we will use to store our instance of the TensorFlow Object
+     * Detection engine.
+     */
+    protected TFObjectDetector tfod;
+
+    @Override
+    public void runOpMode() {
+        // The TFObjectDetector uses the camera frames from the VuforiaLocalizer, so we create that
+        // first.
+        initVuforia();
+        initTfod();
+
+        /**
+         * Activate TensorFlow Object Detection before we wait for the start command.
+         * Do it here so that the Camera Stream window will have the TensorFlow annotations visible.
+         **/
+        if (tfod != null) {
+            tfod.activate();
+
+            // The TensorFlow software will scale the input images from the camera to a lower resolution.
+            // This can result in lower detection accuracy at longer distances (> 55cm or 22").
+            // If your target is at distance greater than 50 cm (20") you can increase the magnification value
+            // to artificially zoom in to the center of image.  For best results, the "aspectRatio" argument
+            // should be set to the value of the images used to create the TensorFlow Object Detection model
+            // (typically 16/9).
+            tfod.setZoom(1.0, 16.0/9.0);
+        }
+
+        /** Wait for the game to begin */
+        telemetry.addData(">", "Press Play to start op mode");
+        telemetry.update();
+        waitForStart();
+
+        if (opModeIsActive()) {
```

Comment:
This line is redundant, and can be deleted.

Reply:
You didn't do what @tmicro said.